src/test/java/com/vytrack/pages/VehiclesPage.java

```diff
@@ -16,4 +23,18 @@ public String getTitle(){
         return Titles.VEHICLES.getValue();
     }
 
+    @FindBy (xpath = "//a[starts-with(@title,'Create ')]")
```

Comment:
Could you please move craeteCarBtn and verifyCannotClickCreateCarBtn to commonFeaturePage ?
As we discussed, all the other pages are sharing the same web element